src/database.ts

```diff
@@ -162,3 +217,33 @@ export class Database<D = unknown> {
     throw errorWithCause("DUMP_ERROR", new Error("not implemented yet"));
   }
 }
+
+/**
+ * Take a normalized statement and convert it to a set of Runnables that can be
+ * used in a call to the registry contract.
+ * @param normalized A normalized statement, e.g. what is returned from the parser's normalize function
+ * @returns An Array of Runnables
+ */
+async function normalizedToRunnables(
+  normalized: NormalizedStatement
+): Promise<Runnable[]> {
+  if (normalized.type !== "write" && normalized.type !== "acl") {
+    throw new Error(
+      "converting to runnable is only possible for mutate statements"
```

Comment:
note: the term "mutate" basically affectively means, (update, insert, delete, and modify acl)